src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/TupleTests.cs

```diff
@@ -261,8 +261,8 @@ .maxstack  6
   IL_002f:  ldstr      ""y""
   IL_0034:  call       ""(int A, int B) Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<(int A, int B)>(string)""
   IL_0039:  ldloc.0
-  IL_003a:  stind.ref
-  IL_003b:  ret
+  IL_003a:  stobj      ""System.ValueTuple<int, int>""
```

Comment:
It is really a result of the changes that makes tuples value types regardless whether backing types are errors or not.

---
In reply to: [81645389](https://github.com/dotnet/roslyn/pull/14161#discussion_r81645389) [](ancestors = 81645389,81644864,81644123)